csharp-scenarios/01-Alien-Translator.ipynb

```diff
@@ -0,0 +1,409 @@
+    "dotnet_interactive": {
+     "language": "csharp"
+    },
+    "vscode": {
+     "languageId": "dotnet-interactive.csharp"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "Console.WriteLine(\"Hi extraterrestrials, my name is myName.\");"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The *Console.WriteLine()* [statement](https://docs.microsoft.com/dotnet/csharp/programming-guide/statements-expressions-operators/statements) allows us to write our string to the console. Let’s break this down. \n",
+    "\n",
+    "> * The [Console](https://docs.microsoft.com/dotnet/api/system.console?view=net-6.0) is what you can use to output strings. \n",
+    "> * The [WriteLine() method](https://docs.microsoft.com/dotnet/api/system.console.writeline?view=net-6.0) writes whatever data you specify in the parenthesis to the output. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Using variables with strings\n",
+    "### Tell the aliens more about yourself\n",
+    "Amazing job! Your linguistic partner says you should introduce yourself a bit more so the aliens know what you do. But, she says to only refer to yourself using your name instead of a pronoun like I, because she can’t translate pronouns yet. Replace all the instances of *myName* with your name and hit run. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "dotnet_interactive": {
+     "language": "csharp"
+    },
+    "vscode": {
+     "languageId": "dotnet-interactive.csharp"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "Console.WriteLine(\"Hi extraterrestrials, my name is myName.\"); \n",
+    "Console.WriteLine(\"myName is a researcher.\"); \n",
+    "Console.WriteLine(\"myName studies language.\"); "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Introduce the aliens to others\n",
+    "Great job! Now other members of your research team want to use your code to introduce themselves. But, going through and changing all the *myName*s with the name of each researcher is tedious and is likely to cause a mistake. Notice how in the strings we want to output, the only thing that changes is the myName. One way to simplify our code and reduce the room for error is by using a variable. A **variable** is a symbol you can use to run the same code with different values. \n",
+    "\n",
+    "Instead of replacing every *myName*, let’s create a string variable called name that we can assign to a different name one time and it will update all the myNames throughout the introduction! Now, replace all the instances of *myName* with our new variable, *name*. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "dotnet_interactive": {
+     "language": "csharp"
+    },
+    "vscode": {
+     "languageId": "dotnet-interactive.csharp"
+    }
+   },
+   "outputs": [],
+   "source": [
```

Comment:
This part is a bit confusing as you actually have to add a $ in front of the " and then do {name}. This isnt' explained... we should give an example before this so they can see it and then run it. or modify the writeline to have it